Remotion/Web/Core/UI/Controls/FormGridManager.cs

```diff
@@ -839,35 +839,35 @@ public HtmlTableRow ControlsRow
       /// <include file='..\..\doc\include\UI\Controls\FormGridManager.xml' path='FormGridManager/FormGridRow/LabelsCell/*' />
       public HtmlTableCell LabelsCell
       {
-        get { return _labelsCell; }
+        get { return Assertion.IsNotNull (_labelsCell, "_labelsCell must not be null."); }
       }
 
       /// <summary> The cell containing the controls. </summary>
       /// <include file='..\..\doc\include\UI\Controls\FormGridManager.xml' path='FormGridManager/FormGridRow/ControlsCell/*' />
       public HtmlTableCell ControlsCell
       {
-        get { return _controlsCell; }
+        get { return Assertion.IsNotNull (_controlsCell, "_controlsCell must not be null."); }
       }
```

Comment:
@MichaelKetting In some other conversation we've agreed on making this property not-null. This unfortunately breaks things, please have a look at the following, causing a yellow page in 4 test.

https://github.com/re-motion/Framework/blob/0bb9bdf4fbc8b324fde86d54780a3723c47b16f7/Remotion/Web/Core/UI/Controls/FormGridManager.cs#L2271-L2281

Reply:
gah, okay. so, nullable for now and we make a cleanup-todo (although, probably won't get done ever in FOrmGridManager...)

Reply:
@MichaelKetting  And should I add assertions or `!` for the usages? If I add assertions, should I add argchecks for usages of `dataRow.ControlsCell` when there's a parameter `dataRow`?

Reply:
For FormGridMaanger: I think we should choose the cheapest way to have compiling code that doesn't change the null behavior, given this needs a follow-up later on (if we want to improve nullablity in FormGridManager). We can (should?) add assertions at the call-site when the value is already a null-dereference anyway. All places where we're not sure, problably just get a `!` + TODO or remain nullable. 

Thing is, FormGridManager may get retired in the next major version and there's a ton of icky nullability in here, so I'm double careful about nullability investments.